docs/content/reference/api/github.com/solo-io/gloo/projects/gateway/api/v1/matchable_http_gateway.proto.sk.md

```diff
@@ -66,7 +66,7 @@ and FilterChain configuration (MatchableHttpGateway CR)
 
 | Field | Type | Description |
 | ----- | ---- | ----------- | 
-| `sourcePrefixRanges` | [[]solo.io.envoy.config.core.v3.CidrRange](../../../../gloo/api/external/envoy/config/core/v3/address.proto.sk/#cidrrange) | CidrRange specifies an IP Address and a prefix length to construct the subnet mask for a CIDR range. See https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#envoy-v3-api-msg-config-core-v3-cidrrange. |
```

Comment:
Is a change like this safe? 

This CidrRange API is part of our public MatchableHttpGateway API, right? The fact that the API is an attempted mirror of an Envoy API is an implementation detail, but I would image we would still want users to have a link to the proto definition. 

Am I missing something?